sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -180,7 +180,7 @@ def __repr__(self):
         )[:1024]
 
 
-class USReceipt(object):  # pylint: disable=too-many-instance-attributes
+class USReceipt(RecognizedForm):  # pylint: disable=too-many-instance-attributes
```

Comment:
Alternatively, instead of inheritng from `RecognizedForm` we can pass in a recognizedForm as a variable and un-flatten this.
```
self.fields = kwargs.get("fields", None)	
self.page_range = kwargs.get("page_range", None)	
self.pages = kwargs.get("pages", None)	
self.form_type = kwargs.get("form_type", None)
```

will be replaced by 
```
self.recognized_form = kwargs.get("recognized_form ", None)	# type: ~models.RecognizedForm
```

Reply:
@johanste thoughts?

Reply:
I do remember @kristapratico had thoughts about making people access fields, pages etc through recognized_form, I can't fully remember the details though.

I think I'm slightly more in favor of adding recognized_form myself, since fields, form_type etc aren't commonly called on a USReceipt, but my feelings aren't particularly strong.